src/test/java/com/cm/businessdirectory/domain/BusinessTest.java

```diff
@@ -0,0 +1,21 @@
+package com.cm.businessdirectory.domain;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class BusinessTest
+{
+    @Test
+    public void givenValidNameAndAddress_whenGetNameAndAddress_thenExpectedValuesReturned() {
```

Comment:
Modify the method name

Reply:
solucionado may.

Reply:
Mejor ponerle al comienzo "givenValidData" que da sentido y contexto